src/files/file.ts

```diff
@@ -0,0 +1,33 @@
+import {Component, Input, ChangeDetectionStrategy, ElementRef, Renderer2, ContentChild } from '@angular/core';
+import {NglFileActionsTemplate} from './templates';
+
+@Component({
+  selector: 'ngl-file',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  templateUrl: './file.pug',
+  host: {
+    '[class.slds-file]': 'true',
+    '[class.slds-file_card]': 'true',
+  },
+})
+export class NglFile {
+  @Input('nglTitle') title: string = null;
```

Comment:
We usually use the `ngl` prefix only in directives, we are going to revisit our naming convention, but can you please remove the prefix from those @Inputs :slightly_smiling_face: ?

Reply:
sure thing!